src/modals/TransactionDetailsModal.tsx

```diff
@@ -153,16 +153,16 @@ const TransactionDetailsModal = ({ transaction, address, onClose }: TransactionD
           <DetailsRow label={t`Inputs`}>
             <AddressList>
               {transaction.inputs?.map((input) => (
-                <ActionLinkStyled key={`${input.outputRef.key}`} onClick={() => handleShowAddress(input.address)}>
-                  <AddressEllipsed key={`${input.outputRef.key}`} addressHash={input.address} />
+                <ActionLinkStyled key={`${input.outputRef.key}`} onClick={() => handleShowAddress(input.address ?? '')}>
+                  <AddressEllipsed key={`${input.outputRef.key}`} addressHash={input.address ?? ''} />
                 </ActionLinkStyled>
               ))}
             </AddressList>
           </DetailsRow>
           <DetailsRow label={t`Outputs`}>
             <AddressList>
               {transaction.outputs?.map((output) => (
-                <ActionLinkStyled key={`${output.key}`} onClick={() => handleShowAddress(output.address)}>
+                <ActionLinkStyled key={`${output.key}`} onClick={() => handleShowAddress(output.address ?? '')}>
```

Comment:
This change doesn't seem to be necessary

Reply:
Same reason as input?.address

Reply:
I think you're wrong, `output.address` is always a `string`.

Reply:
The issue is `output` can be undefined - I'll do the same suggestion as with `input`